detector:
  evt_subsample_fac: 1
  buffer_size_s: 60.0
  tsteps_per_msg: 3
  msg_per_window: 100
  centroid_find_window: 9
  centroid_find_thresh: 7
  triangulation_sv_thresh: 0.005
  min_baseline_dist: 1.0
  max_reproj_err_px: 9
  dist_thresh:
    min: 0.0
    max: 50.0

hough1:
  window_size: 400
  threshold: 15
  nms_radius: 10
  radial_resolution: 670
  angular_resolution: 11
  angle_range:
    min: -5
    max: 5

tracker:
  linearity_window: 9
  linearity_tol_px: 5

tracker_mgr:
  centr_buffer_l: 300
  spawn_thresh: 12
  dxdt_cluster_tol_px: 7
  max_dx_allowed: 100
  dxdt_range:
    min: 50.0
    max: 8000.0
  maturity_age: 0.7

output:
  map_file: ""
  rviz: true
